kb-fonts/package.json

```diff
@@ -14,14 +26,11 @@
     "kb",
     "fonts"
   ],
-
-  "dependencies": {
-    "lit": "^2.6.1"
-  },
-  "author": "",
+  "author": "Royal Danish library",
   "license": "ISC",
```

Comment:
Is this the correct license for us?

Reply:
I don't know much about licenses, but this was suggested but 'npm init' and I didn't change it.
What do you suggest?